### ISSUE TITLE
r3dfox 131 problem in Windows 7

### ISSUE THREAD

**Issue by SUGHuser** [Eclipse-Community/r3dfox-old#205]
If very often opening recieved images in messengers and press ctrl+s for save and ctrl+w for close tab , play music in WMP and use Total Commander approximately per 520-550 saves was will be flicker screen and between r3dfox and WMP , also , in task bar disappear icons see attachment as example
![Screenshot_3428](https://github.com/user-attachments/assets/f9f72a59-3d74-4da9-b1ec-5f99168a9a30)
In version 129 there error absent

**Comment by SUGHuser** [comment]
This bug will be repeat if work 5-7 hours in browser and Total Commander and hearing music
But if I do close explorer and retry open explorer - bug will disappear , but as so this bug , I want what you fix this) I think , what this linked with methods perview on tabs in new version browser , but if disable previews , bug will remain
This in here https://github.com/Eclipse-Community/r3dfox/releases/tag/v131.0.3 , in this version i have such strange bug
I'm not use aero styles , because me this versions not want and I prefer basic style )

**Comment by SUGHuser** [comment]
And so , I rollback to here https://github.com/Eclipse-Community/r3dfox/releases/tag/v129.0.2 , and I work  11 hours with browser and I can not detect problems with flickering screen or lost objects in  task bar ) because I think , what need fix this problem in 131 version
![Screenshot_3431](https://github.com/user-attachments/assets/e21c6cfe-b519-461b-af60-879e10253c0c)
see attachment

**Comment by the-r3dacted** [comment]
I use Windows 7 and I haven't seen any issues like this on it with any version. Nor am I getting any other reports. This is probably something I won't be able to fix.

**Comment by SUGHuser** [comment]
What need for you can fix this bug ? I use plugins in list attachment  , but only 131.0.3 i have problem , maybe in 132 this bug absent , I not tested rc versions.
![Screenshot_3433](https://github.com/user-attachments/assets/266aa889-3b02-4733-9331-7480e0c3ba50)

**Comment by the-r3dacted** [comment]
I'd kinda need to be able to replicate it, then bisect to figure out the cause which takes a whole day if I'm being efficient and don't fuck it up like I usually do.

**Comment by SUGHuser** [comment]
I'm migrated from Firefox 115 to R3DFox 129 and from 129.0.2 I'm trying migrate to 131.0.3.
What new in 131.0.3 if compare with 129.0.2 ? I can seeing changelog ?
I'm think , what this linked with options for previews if put the mouse pointer to the tab and recive preview on tabs , but I think , what this linked with mechanism which use for this , maybe if disable this option , all be work normally , if you not hard , try make 131.0.3 with absent this mechanism for test )
I say not disable in about:config , I say about full delete this mechanism from code browser

**Comment by the-r3dacted** [comment]
> but I think , what this linked with mechanism which use for this , maybe if disable this option , all be work normally , if you not hard , try make 131.0.3 with absent this mechanism for test

Go to about:config and disable browser.tabs.hoverPreview.enabled

**Comment by SUGHuser** [comment]
This not helped, I think what all very deep , before write here , I try disable in config
I'm trying set as in attchment and if compare sets in 129.0.2 
![Screenshot_3435](https://github.com/user-attachments/assets/96630f43-3354-4c95-ae99-d5ed5c416c83)
but for do 131.0.3 as 129.0.2 , me need set so
![Screenshot_3420](https://github.com/user-attachments/assets/1018a48b-bed1-4d47-ba40-d49d8ae119cf)
maybe problem in this , I not known  because asked
I'm say about this 
![Screenshot_3436](https://github.com/user-attachments/assets/7e39ebcc-a594-49e6-819f-889624f61279)
in 129.0.2 this not work and as I understand , this mechanism absent in 129.0.2
And this the only difference with versions 129 -> 130 and 131 , that's why I offer temporary disable this functional from in program code simply for test )
I'm agree be tester version 131 without this function )

**Comment by SUGHuser** [comment]
I'm use R3DFox 129.0.2  a few days and not recive this problem about which I sayd here , I think , what need do fix 131 version and later , because if this problem appeared in 131 version , so this can be migrated problem from version to version in 131 and later )
I hope for understanding

**Comment by the-r3dacted** [comment]
By the way this function exists as far back as 128 so I doubt this is the issue here.

**Comment by SUGHuser** [comment]
Yes , this funciton exist in 128 version , but this function for some reason not work in version 128 and 129 versions , and 129 version work very stable

**Comment by the-r3dacted** [comment]
It works fine for me in 128

**Comment by SUGHuser** [comment]
Thank you , yes , this working in 128 ESR ) I thought earlier what this new feature which appeared in 131 version and next

**Comment by the-r3dacted** [comment]
Looking a bit more carefully at this, you seem to be running the browser on a very old system. This feels like a GPU driver incompatibility that occurs due to some changes to a component in a later release.

Not sure I'd be able to fix this alone. Although it might be worth seeing if Mozilla fixes it in later releases.

**Comment by the-r3dacted** [comment]
Anything new? Is it still an issue in 134?

**Comment by SUGHuser** [comment]
I can test new version and report to you )

**Comment by SUGHuser** [comment]
Setting Enable caption bar fog not work , if I cher or unchek this parameter , nothing not will not change

**Comment by SUGHuser** [comment]
Also bug if use Simple Tab Gropups , if I want move tab to other group , absent menu with actions

**Comment by the-r3dacted** [comment]
> Setting Enable caption bar fog not work , if I cher or unchek this parameter , nothing not will not change

According to your previous screenshots, you seem to be in basic mode. Aero fog isn't meant for basic mode, therefore it shouldn't do anything there. I'm not sure why you would want it here.

If you really want it, maybe try going to about:config and making a boolean named widget.windows-style.show-fog set to true. That might do it.

> Also bug if use Simple Tab Gropups , if I want move tab to other group , absent menu with actions

I'm not sure what this means? Could you send a video of what it should look like, and what this issue is?

**Comment by SUGHuser** [comment]
If I use  widget.windows-style.show-fog set to true. This give that  so color theme from not like that as I got used to it

This 129
![Image](https://github.com/user-attachments/assets/2f50409d-b653-4c77-a1b6-c80e90ea9cef)

This 134
![Image](https://github.com/user-attachments/assets/2738f21f-8108-48ce-9715-5cee5481640a)

This settings
![Image](https://github.com/user-attachments/assets/cae1367b-3db0-48ef-83d9-9e77edced4d9)

this because not work r3dfox.less.aero.fog , as in settings also in about:config in 129 this option work normal )I tested on 134.0.3
and latest same

**Comment by the-r3dacted** [comment]
Ok, so I think up to 132 I had it set that Aero fog applied to basic mode, but this is incorrect, basic mode has no need for Aero fog, so when the theme was redone for 133, I had corrected it and made it only apply with Aero, and a special classic mode fog that was in 115 but didn't seem to work. I think I also did this so XP OCAPI users would not have fog with Luna which looks especially bad.

I guess I could add it back now since the user would need to manually enable it with the new theme.

**Comment by SUGHuser** [comment]
You this sayd seriously ?
In what way you do work in XP if I get message what this application not win32 if run in Windows XP ?

**Comment by the-r3dacted** [comment]
> In what way you do work in XP if I get message what this application not win32 if run in Windows XP ?

Currently OCAPI or One Core API is needed for use on XP.
https://github.com/shorthorn-project/One-Core-API-Binaries

**Comment by SUGHuser** [comment]
Thank you ) I understand )
But I think what in x64 versions better do as in  FF 115 and similar )
Or separating settings for Windows XP and Windows Vista and newer )

**Comment by SUGHuser** [comment]
I all understanded , If I set Modern UI (Requires browser restart) all normaly , but if I uncheck this , I have all same problems )

![Image](https://github.com/user-attachments/assets/573b8aba-4657-4a2a-8390-02c69056fea0)

I'm test this on version https://github.com/Eclipse-Community/r3dfox/releases/tag/v135.0.1-1-hotfix

**Comment by the-r3dacted** [comment]
From what I grasp the issue this issue was created for was fixed? So I'll close this then? If what was mentioned in the first post is still an issue, let me know and I'll reopen. However if something else like the fog is an issue, make a new issue for it.
